Add three presets for `substation=minor_distribution` (Kiosk, Building, Tower)

tordans opened the pull request:
### Description, Motivation & Context

I am having a hard time mapping those minor distribution "buildings". This PR adds three presets that provide explicit tagging recommendation for common objects.

**Relevant OSM Wiki links:**
- https://wiki.openstreetmap.org/wiki/Tag:substation%3Dminor_distribution
- https://wiki.openstreetmap.org/wiki/Tag:power%3Dsubstation

**Examples:**
- Well mapped: https://www.openstreetmap.org/way/872220810
- Insufficiently mapped: https://www.openstreetmap.org/node/7721504279
- Probably wrongly mapped: https://www.openstreetmap.org/way/797976482


**Relevant tag usage stats:**

None; Hard to analyse given the many sub tags.

In general all those tags are well documented and established.
https://taghistory.raifer.tech/#***/building/service&***/building/transformer_tower&***/substation/minor_distribution&***/man_made/street_cabinet&***/power/substation

### Related issues

None

### Translations

Some keywords we should add to the German translation:

**`substation_minor_distribution_building.json`**

> stromhäuschen, stromhaus, trafostation, Strom

```
    "tags": {        "building": "service",        "utility": "power"    },
    "addTags": {        "power": "substation",        "substation": "minor_distribution"    },
    "terms": [        "minor distribution station",        "small masonry building"    ],
    "name": "Substation (Building)"
```

**`substation_minor_distribution_kiosk.json`**

> Trafo-Kompaktstation, Stromhäuschen, Stromhaus, Strom

```
    "tags": {        "man_made": "street_cabinet",        "substation": "minor_distribution"    },
    "addTags": {        "utility": "power",        "power": "substation"    },
    "terms": [        "kiosk",        "compact"    ],
    "name": "Substation (Kiosk)"
```


**`substation_minor_distribution_tower.json`**

> Turm, Turmstation, Trafotrum, Transformatorenturm, Trafoturmstation, Trafo-Kompaktstation, Strom

```
    "tags": {        "building": "transformer_tower"    },
    "addTags": {        "utility": "power",        "power": "substation",        "substation": "minor_distribution"    },
    "terms": [        "transformer building",        "electricity pole"    ],
    "name": "Substation (Tower)"
```

### Checklist and Test-Documentation Template

<details><summary>Read on to get your PR merged faster…</summary>

Follow these steps to test your PR yourself and make it a lot easier and faster for maintainers to check and approve it.

**This is how it works:**
1. After you submit your PR, the system will create a preview and comment on your PR:
   > 🍱 You can preview the tagging presets of this pull request here.
   If this is your first contribution to this project, the preview will not happen right away but requires a click from one of the project members. We will do this ASAP.

2. Once the preview is ready, use it to test your changes.

3. Now copy the snippet below into a new comment and fill out the blanks.

4. Now your PR is ready to be reviewed.

```
## Test-Documentation

### Preview links & Sidebar Screenshots

<!-- Use the preview to find examples, select the feature in question and **copy this link here**.
     Find examples of nodes/areas. Find examples with a lot of tags or very few tags. – Whatever helps to test this thoroughly.
     Add relevant **screenshots** of the sidebar of those examples. -->

<!-- FYI: What we will check:
     - Is the [icon](https://github.com/ideditor/schema-builder/blob/main/ICONS.md) well chosen.
     - Are the fields well-structured and have good labels.
     - Do the dropdowns (etc.) work well and show helpful data. -->

### Search

<!-- **Test the search** of your preset and share relevant **screenshots** here.
     - Test the preset name as search terms.
     - Also test the preset terms and aliases as search terms (if present). -->

### Info-`i`

<!-- **Test the info-i** for your fields and preset and share relevant **screenshots** here.
     The info needs to help mappers understand the preset and when to use it.
     [Learn more…](https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md#info-i)
 -->

### Wording

- [ ] American English
- [ ] `name`, `aliases` (if present) use Title Case
- [ ] `terms` (if present) use lower case, sorted A-Z
<!-- Learn more in https://github.com/openstreetmap/id-tagging-schema/blob/main/GUIDELINES.md#2-design-the-preset -->
```

</details>

tordans committed:
Add three presets for substation=minor_distribution (Kiosk, Building, Tower)

https://wiki.openstreetmap.org/wiki/Tag:substation%3Dminor_distribution

tordans committed:
Add all tags to `addTags`

This will hopefully make it, so that when I select a building=yes this gets overwritten with building=service.

tordans committed:
Remove "power" field

This looked confusing in the editor, see testing notes

tordans committed:
Remove some fields that no one can or should change for this preset

tordans commented:
## Test-Documentation `substation_minor_distribution_building.json`

### Preview links & Sidebar Screenshots

Overpass: https://overpass-turbo.eu/s/23Ui

- https://pr-1543--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/50.98226/12.45687&disable_features=boundaries&background=Bing&id=w1373448582

> (Outdated; Fields changed a bit)
> <img width="300" alt="image" src="https://github.com/user-attachments/assets/eddd94c5-bfd2-4cd4-b305-04bf8cb888c6" />

- The "Type", "Type" Fields are not ideal. I remove the first Type which was `power=substation` from the fields to clean this up. I don't think that field needs to be present given that the preset is named Substation and the options don't show anything I would like to switch to.

- I also remove the "Type" (`substation`) because it does not explain anything and no one would want to change that (or should)

- FYI: When I select an existing building and change the preset, all fields get updated.

### Search

> <img width="300" alt="image" src="https://github.com/user-attachments/assets/c5e5bcd4-7f56-43af-96b3-ba718aa621aa" />

Search terms work great.

### Info-`i`

Picket the building data here to signify that this has to be an actual building, not the kiosk type.

- https://wiki.openstreetmap.org/wiki/Item:Q5341
- https://wiki.openstreetmap.org/wiki/Tag:building=service
> <img width="367" alt="image" src="https://github.com/user-attachments/assets/13027f5c-b225-4364-8678-6bcef75dbacf" />


---

## Test-Documentation `substation_minor_distribution_tower.json`

### Preview links & Sidebar Screenshots

Overpass: https://overpass-turbo.eu/s/23Uf

Example: https://pr-1543--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/51.05082/12.82381&disable_features=boundaries&background=Bing&id=w228749027

> (Outdated; Fields changed a bit)
> <img width="300" alt="image" src="https://github.com/user-attachments/assets/74b6a4d3-ccf0-43c8-9447-a262db6f2eac" />

- Note: The wiki page https://wiki.openstreetmap.org/wiki/Tag:building%3Dtransformer_tower does not mention but the page https://wiki.openstreetmap.org/wiki/Tag%3Asubstation%3Dminor_distribution does and I think it makes sense to have this tag on all those features.



### Search

./.

### Info-`i`

- https://wiki.openstreetmap.org/wiki/Tag:building%3Dtransformer_tower
- https://wiki.openstreetmap.org/wiki/Item:Q6553
> <img width="365" alt="image" src="https://github.com/user-attachments/assets/8a46dbd0-8563-481b-b056-9869db4ca48e" />




---

## Test-Documentation `substation_minor_distribution_kiosk.json`

### Preview links & Sidebar Screenshots

Overpass: https://overpass-turbo.eu/s/23Ug

Example: https://pr-1543--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/51.04404/12.80905&disable_features=boundaries&background=Bing&id=n6544854454

> (Outdated; Fields changed a bit)
> <img width="300" alt="image" src="https://github.com/user-attachments/assets/1196f77c-c219-425d-8da9-90752dc09e75" />

- I removed the "Type (`man_made`)", because it does not explain anything and no one would want to change that (or should)
- I also remove the "Type" (`substation`) for the same reason


### Search

./.

### Info-`i`

- https://wiki.openstreetmap.org/wiki/Tag:substation%3Dminor_distribution
- https://wiki.openstreetmap.org/wiki/Item:Q6523
> <img width="367" alt="image" src="https://github.com/user-attachments/assets/128f1770-b84d-486c-9b3d-1adc1744b315" />

tordans commented:
@flacombe could you have a look at this and check if this has any relevant issues?

flacombe commented:
Hello @tordans 

Thank you for mentioning and refining those presets!

I see no major problems:
* Tagging is fine
* Willing to remove unnecessary static fields from the presets is relevant

The two points I want to raise are:
* there is no need for frequency on those substations. It's more a property for devices (transformers, compensators inside substations) than for facilities.
* A fourth preset for substations on poles can be valuable in some countries (some countries consider a transformer on a pole is a minor distribution substation and some other don't)

Furthermore, although it's advanced tagging so probably outside of the scope of these presets, cabinet tagging may be completed with transformers inside of them.
Charts can be seen here: https://wiki.openstreetmap.org/wiki/Proposal:Substation_nodes_extension

All the best

tordans commented on data/presets/power/substation_minor_distribution_building.json:
```diff
@@ -0,0 +1,40 @@
+        "address",
+        "gnis/feature_id-US",
+        "transformer",
+        "phases",
+        "frequency"
```
```suggestion
        "phases"
```

tordans commented on data/presets/power/substation_minor_distribution_kiosk.json:
```diff
@@ -0,0 +1,40 @@
+        "gnis/feature_id-US",
+        "transformer",
+        "phases",
+        "frequency",
```
```suggestion
```

tordans committed:
Apply suggestions from code review: Remove `frequency`

https://github.com/openstreetmap/id-tagging-schema/pull/1543#issuecomment-3194448040

tordans commented:
Thanks for checking this!

> there is no need for frequency on those substations. It's more a property for devices (transformers, compensators inside substations) than for facilities.

Thanks. I [saw it on one example (that fits the image I have in mind for this one of the presets)](https://wiki.openstreetmap.org/wiki/Tag:substation%3Dminor_distribution) but removed it now.

> A fourth preset for substations on poles can be valuable in some countries (some countries consider a transformer on a pole is a minor distribution substation and some other don't)

Given the quite complex tagging I think adding more presets would be a good idea. Maybe some should be scoped for certain countries… — But lets do that later/separately. And ideally by someone who knows more about this than me ;-).

FloEdelmann commented on data/presets/power/substation_minor_distribution_kiosk.json:
```diff
@@ -0,0 +1,39 @@
+    },
+    "terms": [
+        "kiosk",
+        "compact"
```
```suggestion
        "compact",
        "kiosk"
```

tordans committed:
Apply suggestion from @FloEdelmann

Co-authored-by: Flo Edelmann <git@flo-edelmann.de>

tordans committed:
Apply suggestion from @FloEdelmann

Co-authored-by: Flo Edelmann <git@flo-edelmann.de>

matkoniecz reviewed (changes requested):
https://github.com/openstreetmap/id-tagging-schema/pull/1543#discussion_r2452094679 

(not saying that suggestion/request present there must be followed - but can you answer it?)

matkoniecz commented on data/presets/power/substation_minor_distribution_building.json:
```diff
@@ -0,0 +1,39 @@
+        "address",
+        "gnis/feature_id-US",
+        "transformer",
+        "phases"
```
@flacombe I am a bit confused and I wrote about it at https://wiki.openstreetmap.org/w/index.php?title=Talk%3ATag%3Asubstation%3Dminor_distribution#points_for_electricity_stuff - when it is supposed to be on a separate point?

flacombe replied:
Hello, following my answer on wiki, this proposal is the most comprehensive to answer this question: https://wiki.openstreetmap.org/wiki/Proposal:Substation_nodes_extension

Mapping on nodes is required when you don't have any polygon to put the substation onto (for instance for cabinets drawn as nodes and in larger buildings, like we do for shops)

flacombe commented on data/presets/power/substation_minor_distribution_kiosk.json:
```diff
@@ -0,0 +1,39 @@
+    },
+    "addTags": {
+        "man_made": "street_cabinet",
+        "substation": "minor_distribution",
```
That's not the only value can be found on a cabinet: distribution can also be found on cabinets.
